sycl/test-e2e/Adapters/sycl-ls-gpu-cuda.cpp

```diff
@@ -1,8 +1,7 @@
 // REQUIRES: gpu, cuda
-// REQUIRES: build-and-run-mode
 
-// RUN: env ONEAPI_DEVICE_SELECTOR="cuda:*" sycl-ls --verbose >%t.cuda.out
-// RUN: FileCheck %s --check-prefixes=CHECK-BUILTIN-GPU-CUDA,CHECK-CUSTOM-GPU-CUDA --input-file %t.cuda.out
+// RUN: %if run-mode %{ env ONEAPI_DEVICE_SELECTOR="cuda:*" sycl-ls --verbose >%t.cuda.out %}
```

Comment:
I think we need to use `%{run-unfiltered-devices}` here and elsewhere. That would matter if the suite would run on some kind of simulator/emulator using `run_launcher` parameter.